app/javascript/components/App.js

```diff
@@ -41,6 +42,7 @@ class App extends Component {
               component={SignUp}
             />
           }
+          <Route path='*' component={MissionPage} />
```

Comment:
I think you would want your Mission Page to have a route since you have a NavLink in your Navigation component that is referencing a specific path.